cleaner.py

```diff
@@ -3,7 +3,7 @@
 
 
 party_recode = {
-    "Americans Elect": "AME",	
```

Comment:
My editor gets rid of this trailing whitespace automatically

Reply:
No problem with removing trailing whitespace, but I don't like removing the indents from the blank lines below.